Dockerfile

```diff
@@ -767,24 +767,6 @@ RUN cargo build --release --locked
 
 # =^..^=   =^..^=   =^..^=   =^..^=   =^..^=   =^..^=   =^..^=   =^..^=   =^..^=
 
-FROM sdk-cargo AS sdk-cargo-make
```

Comment:
We unnecessarily depend on `cargo-make`'s license being present in twoliter.. I've cut https://github.com/bottlerocket-os/twoliter/issues/458 to stop depending on it. That need not block this change.